src/aat/resources/features/F-050/F-050.feature

```diff
@@ -0,0 +1,58 @@
+@F-050
+Feature: Validate a set of fields as Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-301
+  Scenario: must validate when all fields are valid
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [uses the Case ID of the case just created]
+    And it is submitted to call the [validation of a set of fields as Case worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [has the 200 return code]
+    And the response has all the details as expected
+
+  @Ignore
+  #To be completed on https://tools.hmcts.net/jira/browse/RDM-6654
+  @S-297
+  Scenario: must not validate when CMC ExternalID is not unique / already exists
+    Given a user with a detailed profile in CCD
```

Comment:
This won't work.

Reply:
Updated this to : Scenario: must not validate when CMC ExternalID is not unique or already exists.  This is an @Ignored scenario and will implement later.